apps/application-system/api/src/app/modules/payment/utils/findItemType.ts

```diff
@@ -0,0 +1,74 @@
+//import { DrivingLicenseType } from '../types/schema'
+
+export const findItemType = (itemCode: string) => {
+  let typeId = itemCode
+  // AYXXX PayableDummyTemplate should be deleted when Dummy testing has finished.
+  switch (typeId) {
+    case 'AYXXX':
+      typeId = 'PayableDummyTemplate'
```

Comment:
it might be more elegant to just return the string directly, instead of assigning to a variable name

Reply:
one-entry-one-exit vs elegance.
Both can be argued for.

Reply:
one-entry-one-exit is an anti-pattern, imo, unless you are dealing with a situation where you need to clean up before exiting a function. readability should be the primary concern.

>“Programs must be written for people to read, and only incidentally for machines to execute.”

😃